CMakeLists.txt

```diff
@@ -193,8 +193,8 @@ if(USE_PREBUILT_MESAPY)
 add_custom_command(
   OUTPUT ${MESAPY_OUTPUTS}
   COMMAND
-    wget -qN https://mesapy.org/release/${MESAPY_VERSION}-mesapy-sgx.tar.gz &&
-    tar xzf ${MESAPY_VERSION}-mesapy-sgx.tar.gz
+    wget -qN https://github.com/mesatee/mesapy/releases/download/${MESAPY_VERSION}/mesapy-${MESAPY_VERSION}.tar.gz &&
```

Comment:
I tried to open this URL, but got 404. Please double check it, thanks.

Reply:
Can you post your URL here? The URL here is https://github.com/mesatee/mesapy/releases/download/v1.0.0-sgx/mesapy-v1.0.0-sgx.tar.gz after `MESAPY_VERSION` is filled.

Reply:
@mssun will help to find a link for this PR, thanks.